doc/contribute/onboarding.rst

```diff
@@ -0,0 +1,102 @@
+Onboarding
+==========
+
+This page aims to be a list of resources that should be helpful if you want to start contributing to CCF.
+
+Introduction
+------------
+
+Start by reading :doc:`/overview/what_is_ccf`.
+
+If you encounter any terms or acronyms that you don't know, check the :doc:`/overview/glossary`. If the word you are looking for isn't defined, create an `issue <https://github.com/microsoft/CCF/issues/new/choose>`_.
```

Comment:
Even though this page is more informal than the rest of the documentation, I'd avoid using abbreviations (e.g. "don't")?

Reply:
Ok, addressed, there's a few other places where we've been doing this.